### PR TITLE
Add hap blog post

### PR DESCRIPTION
**Pull request by leavittx**
I'm using links to images hosted on github

### CODE REVIEW

**Comment by leavittx** [comment]
Test comment

**Comment by antongit** [comment]
Test comment

**Review by joreg — CHANGES_REQUESTED** [review]

File: content/blog/2022/other/news_in_hap_world.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+---
+draft: false
+date: "2022-04-12"
+title: "News in Hap world"
+description: "The Hap player news"
+categories: User Contribution
+author: lev
+---
+
+There were multiple important things happening in the Hap GPU-accelerated video playback codec world recently. Sadly, there is no a single website with all these updates (though there is an [official website](http://hap.video). I will try to put the information all together in this blog post.
```
Comment (joreg) [inline content/blog/2022/other/news_in_hap_world.md:11]: _not_ a single website..

File: content/blog/2022/other/news_in_hap_world.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The support for encoding was shortly added to the amazing [Jokyo Hap Encoder](https://jokyohapencoder.com/) — which is another recent addition to the Hap ecosystem. You can use it for either Adobe Media Encoder or for After Effects. It's nice not only because it's the first and the only one to support Hap R, also it improves Hap and Hap Q quality compared to the community encoders.  Hopefully, GPU accelerated Hap R encoding will be available somewhere soon and then it'll be possible to quickly prepare new footage versions for playback.
+
+Demolition Media Hap Player for vvvv beta/gamma supports it already for some time, and you can use Hap R in both RGB and RGBA modes.
+
+### Hap player for vvvv updates
+Next, I'd like to list what new features were added to the vvvv's Demolition Media Hap Player:
+1. The one I especially like — texture updates are done now in a background thread — main thread is unaffected by playback
+2. The loop playback has became seamless (thanks to pre-caching next frames in a non-stop manner). No hiccups at the start of new loop whether you have custom start/end frames set or not
```
Comment (joreg) [inline content/blog/2022/other/news_in_hap_world.md:39]: has _become_ seamless

File: content/blog/2022/other/news_in_hap_world.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+I'm not removing the old plugins from anywhere, they are still downloadable (for non-commercial usage), but I really encourage interested people to go to Patreon as I will post new updates and fixes there (already posted first portion of them last week).
+
+[![Become a Patron](https://raw.githubusercontent.com/leavittx/leavittx.github.io/master/hap_blog_post_2022/images/becomeapatron.jpg)](https://www.patreon.com/hap_player)
+
+
+If you feel uncomfortable with Patreon for some reason, please  [get in contact](mailto:lev.panov@gmail.com) and we'll figure something out. 
+
+**It's possible to send invoices on a regular basis -- just drop me a message after joining on Patreon.** Also feel free to share you thoughts in the comments, regarding tiers, or anything. 
+
+Update: for users unable to use Patreon there is a [Boosty page](https://boosty.to/hap_player) now.
```
Comment (joreg) [inline content/blog/2022/other/news_in_hap_world.md:70]: Consider removing the "Update" because it is anyway the first post now?!

**Review by joreg — CHANGES_REQUESTED** [review]

File: content/blog/2022/other/news_in_hap_world.md (hunk 2 above)
Comment (leavittx) [reply content/blog/2022/other/news_in_hap_world.md:39]: oh, right

**Review by joreg — APPROVED** [review]

**Pull request merged**